src/js/config/config.js

```diff
@@ -15,5 +15,10 @@ var config = {
 };
 if (config.environment === "production") {
   config.rootUrl = "dist/";
+} else {
+  try {
+    var configDev = require("./config.dev");
+    config = Object.assign(config, configDev);
+  } catch (e) {}
```

Comment:
Thanks for the catch. I will 

1. explicity add the `no-empty` rule to our `.eslintrc` file as it's apparently turned on implicitly, and 
2. add a ` // Do nothing` comment within the `catch`